project_access_tokens.go

```diff
@@ -34,15 +34,16 @@ type ProjectAccessTokensService struct {
 //
 // GitLab API docs: https://docs.gitlab.com/ee/api/resource_access_tokens.html
 type ProjectAccessToken struct {
-	ID        int        `json:"id"`
-	UserID    int        `json:"user_id"`
-	Name      string     `json:"name"`
-	Scopes    []string   `json:"scopes"`
-	CreatedAt *time.Time `json:"created_at"`
-	ExpiresAt *ISOTime   `json:"expires_at"`
-	Active    bool       `json:"active"`
-	Revoked   bool       `json:"revoked"`
-	Token     string     `json:"token"`
+	ID          int        `json:"id"`
+	UserID      int        `json:"user_id"`
+	Name        string     `json:"name"`
+	Scopes      []string   `json:"scopes"`
+	CreatedAt   *time.Time `json:"created_at"`
+	ExpiresAt   *ISOTime   `json:"expires_at"`
+	Active      bool       `json:"active"`
+	Revoked     bool       `json:"revoked"`
+	Token       string     `json:"token"`
+	AccessLevel int        `json:"access_level"`
```

Comment:
Please use [this](https://github.com/xanzy/go-gitlab/blob/master/types.go#L54) one instead of an `int`.